source/extensions/tracers/common/ot/opentracing_driver_impl.cc

```diff
@@ -45,19 +45,13 @@ class OpenTracingHTTPHeadersReader : public opentracing::HTTPHeadersReader {
   // opentracing::HTTPHeadersReader
   opentracing::expected<opentracing::string_view>
   LookupKey(opentracing::string_view key) const override {
-    const Http::HeaderEntry* entry;
-    Http::HeaderMap::Lookup lookup_result =
-        request_headers_.lookup(Http::LowerCaseString{key}, &entry);
-    switch (lookup_result) {
-    case Http::HeaderMap::Lookup::Found:
+    const Http::HeaderEntry* entry = request_headers_.get(Http::LowerCaseString{key});
+    if (entry != nullptr) {
       return opentracing::string_view{entry->value().getStringView().data(),
                                       entry->value().getStringView().length()};
-    case Http::HeaderMap::Lookup::NotFound:
+    } else {
```

Comment:
Is the distinction between non-inline header and not-found header impactful?

Reply:
No, I don't think it matters for this case. They just want the header.